common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -16,8 +16,16 @@
 #include <bcl/network/network_utils.h>
 #include <bcl/son/son_wireless_utils.h>
 #include <easylogging++.h>
+
```

Comment:
Commit message should include the commit hash of the DWPAL library that was used.

Reply:
Are you sure it's the best way to keep compatibility?
Perhaps DWPAL maintainers should increase the library version when introducing breaking changes?

Reply:
What's the standard here? Simply add something like:
DWPAL Library Hash: b1341a600d87bd808e160ef40ccd2cb51db18b2d?

Reply:
Yes, this works

Reply:
Done